lib/commons/dom/is-modal-open.js

```diff
@@ -34,26 +40,32 @@ function isModalOpen(options) {
   // in the viewport. since we aren't sure if it is or is not a modal this is
   // just our best guess of being one based on convention.
 
-  if (cache.get('isModalOpen')) {
-    return cache.get('isModalOpen');
+  if (definitelyModalExists()) {
+    return true;
+  } else if (couldBeModalExists(modalPercent)) {
+    return true;
   }
+  return undefined;
```

Comment:
If we pass undefined into the cache, it's not going to know this has been set. `detectModal` is going to run over and over again.

```suggestion
  return false;
```